sdk/storage/azure-storage-file-share/tests/test_share.py

```diff
@@ -7,6 +7,7 @@
 import time
 import unittest
 from datetime import datetime, timedelta
+from os import access
```

Comment:
Umm.. what? 😄 Assuming this was an accident

Reply:
Wow nice catch! Thanks for putting it out :)